src/Resolve/Resolve.jl

```diff
@@ -411,11 +372,182 @@ function enforce_optimality!(sol::Vector{Int}, graph::Graph)
         staged = staged_next
     end
 
+    # @info "clearing up: $([pkgID(p0, graph) for p0 in findall(uninst) if sol[p0] ≠ spp[p0]])"
```

Comment:
Is this one (and the comment below) intended to be left in (to ease future debugging perhaps)?

Reply:
Ah no sorry, I wasn't intending to leave this one here. I'll clean it up.